### DISCUSSION TITLE
没有问题要问，只想表达对作者的赞美

### DISCUSSION THREAD

**Discussion by mwdotzom** [qier222/YesPlayMusic#1021] [General]
看上去只是做了个播放器 实际往夸张说，对精神洁癖用户算得上一种拯救
这样不为回报地投身一件事，像做一件艺术品一样完善它，这样的人是我最景仰的
不说了 nb就完事了

**Comment by qier222** [comment]
有私心的，就是打算骗点stars丰富下简历，没有人用的话早就不维护了😬

**Reply by mwdotzom** [reply]
《 没 有 人 用 》（指170/680 Issues）
